Source/Header/DaySymbolsView.swift

```diff
@@ -3,7 +3,15 @@ import UIKit
 
 public final class DaySymbolsView: UIView {
   public private(set) var daysInWeek = 7
-  private var calendar = Calendar.autoupdatingCurrent
+  public var calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      // TODO: I honestly don't know if I should be doing this. Changes to
```

Comment:
I think, it's a good practice. The API suggest the possibility of changing the Calendar, what if e.g. 1st day of week has changed? In that case reconfiguration would be necessary.